com.github.VescTool.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: com.github.VescTool
```

Comment:
Shouldn't the id be `com.vest_project.VestTool`

Reply:
yep